src/components/Keypad/Keypad.spec.js

```diff
@@ -22,15 +22,36 @@ describe('Keypad', () => {
         expect(subject.find('.numbers-container').length).toBe(1);
     });
 
-    it('renders the keypad values', () => {
-        expect(subject.find('.numbers-container').text()).toEqual('012');
+    it('renders the keys for each of the number values', () => {
+        expect(subject.find('.numbers-container Key').length).toBe(props.numbers.length);
     })
 
-    it('renders the operator values', () => {
-        expect(subject.find('.operators-container').text()).toEqual('+-*/');
-    })
-    
-    it('renders a <Key /> component', () => {
-        expect(subject.find('Key').length).toBe(1)
+    it('renders the keys for each operator value', () => {
+        expect(subject.find('.operators-container Key').length).toBe(props.operators.length)
     })
 });
+
+describe('Mounted Keypad', () => {
+    let subject;
+    let props = {
+        callOperator: jest.fn(),
+        setOperator: jest.fn(),
+        updateDisplay: jest.fn(),
+        numbers: ['0', '1', '2'],
+        operators: ['+', '-', '*', '/'],
+    }
+
+    beforeEach(() => {
+        subject = mount(
+            <Keypad {...props} />
+        );
+    });
+
+    it('should render each operator to the DOM', () => {
```

Comment:
```suggestion
    it('should render each operator', () => {
```
I think "to the DOM" is implied.